fhir-operation-bulkdata/src/main/java/com/ibm/fhir/operation/bulkdata/client/BulkDataClient.java

```diff
@@ -141,7 +141,7 @@ public String submit(MediaType outputFormat, Instant since, List<String> types,
 
         // Need to push this into a property.
         WebTarget target = getWebTarget(properties.get(BulkDataConfigUtil.BATCH_URL));
-        
+
         System.out.println("-> " + properties.get(BulkDataConfigUtil.BATCH_URL));
```

Comment:
Switch to a logger?

Reply:
ha, I didn't notice that System.out is used here, the change was by action of eclipse save. let me change this.

Reply:
I'm not sure that this is removed? Git shows it still visible

Reply:
my view was cached... I had to do a hard refresh